src/functionalTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/GetSasTokenTest.java

```diff
@@ -96,11 +93,11 @@ public void should_throw_exception_when_requested_service_is_not_configured() th
     public void sas_token_should_have_read_and_write_capabilities_for_service() throws Exception {
         String testContainerName = conf.getString("test-storage-container-name");
         String sasToken = testHelper.getSasToken(testContainerName, this.testUrl);
-        CloudBlobContainer testSasContainer =
-            testHelper.getCloudContainer(sasToken, testContainerName, this.blobContainerUrl);
+        BlobContainerClient testSasContainer =
+            testHelper.getContainerClient(sasToken, testContainerName, this.blobContainerUrl);
 
         destZipFilename = testHelper.getRandomFilename();
-        testHelper.uploadAndLeaseZipFile(
+        this.leaseId = testHelper.uploadAndLeaseZipFile(
```

Comment:
we are uploading and after that we are leasing it, we are using random files names.
do we need to lease ?

Reply:
i guess not :)

Reply:
I will move later.